book/src/kubernetes-changelog.md

```diff
@@ -3,6 +3,18 @@
 This page summarizes major CSI changes made in each Kubernetes release. For
 details on individual features, visit the [Features section](features.md).
 
+
+## Kubernetes 1.28
+
+### Features
+* GA
+    * [Automatic retroactive assignment of a default StorageClass](https://kubernetes.io/blog/2023/08/18/retroactive-default-storage-class-ga/)
+    * [Non-graceful node shutdown](https://kubernetes.io/blog/2023/08/16/kubernetes-1-28-non-graceful-node-shutdown-ga/)
+* Removals:
+    * [Removal of CSI Migration for GCE PD](https://github.com/kubernetes/enhancements/issues/1488)
+* Deprecations:
+    * [Ceph RBD in-tree plugin](https://github.com/kubernetes/kubernetes/pull/118303)
+    * [Ceph FS in-tree plugin](https://github.com/kubernetes/kubernetes/pull/118143)
 ## Kubernetes 1.26
```

Comment:
Can you add 1.27 features as well?

Reply:
Added some, might not be exhaustive

Reply:
Add the following:
Beta:
- CSI node expand secret: https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3107-csi-nodeexpandsecret
- Prevent unauthorized volume mode converstion: https://github.com/kubernetes/enhancements/issues/3141

In-tree plugin removal:
- AWE EBS
- Azure Disk